js/main-menu.js

```diff
@@ -0,0 +1,129 @@
+(function ($, _) {
+  $(document).ready(function () {
+    amendMarkupOfMenuItemsWithFontAwesomeIcons();
+    customizeQuickSearchField();
+  });
+
+  /**
+   * Amends the markup of any menu item with a FontAwesome icon
+   */
+  function amendMarkupOfMenuItemsWithFontAwesomeIcons() {
+    menuItemsWithFontAwesomeIcon().each(function () {
+      var $menuItem = jQuery(this);
```

Comment:
I wonder why sometimes `$` used and sometimes` jQuery`?

Reply:
mistake, fixing it